app/src/main/java/com/kks/nimblesurveyjetpackcompose/util/extensions/NetworkExtension.kt

```diff
@@ -45,7 +37,8 @@ suspend fun <T> safeApiCall(
                 is UnknownHostException -> ResourceState.NetworkError
                 is TimeoutCancellationException -> ResourceState.NetworkError
                 is HttpException -> {
-                    val errorResponse = throwable.response()?.parseErrJsonResponse<CustomErrorResponse>()
+                    val errorResponse =
+                        throwable.response()?.parseErrJsonResponse<CustomErrorResponse>()
```

Comment:
Perhaps we can check our codestyle in Android Studio preferences to see if hard wrap is set to `120` 😉 
```suggestion
                    val errorResponse = throwable.response()?.parseErrJsonResponse<CustomErrorResponse>()
```